src/pages/Admin/Venue/VenueDetailsForm.tsx

```diff
@@ -0,0 +1,351 @@
+import { useUser } from "hooks/useUser";
+
+import { PlayaContainer } from "pages/Account/Venue/VenueMapEdition";
+import { VenueDetailsSubForm } from "pages/Admin/Venue/VenueDetailsSubForm";
+
+import "firebase/functions";
+
+import {
+  editVenueCastSchema,
+  validationSchema,
+} from "./DetailsValidationSchema";
+import { WizardPage } from "./VenueWizard";
+
+// @debt refactor any needed styles out of this file (eg. toggles, etc) and into DetailsForm.scss/similar, then remove this import
+import "../Admin.scss";
+import "./Venue.scss";
+
+export type FormValues = Partial<Yup.InferType<typeof validationSchema>>; // bad typing. If not partial, react-hook-forms should force defaultValues to conform to FormInputs but it doesn't
+
+interface DetailsFormProps extends WizardPage {
+  venueId?: string;
+}
+
+const iconPositionFieldName = "iconPosition";
+
+export const VenueDetailsForm: React.FC<DetailsFormProps> = ({
+  previous,
+  state,
+  venueId,
+}) => {
+  const defaultValues = useMemo(
+    () =>
+      venueId
+        ? editVenueCastSchema.cast(state.detailsPage?.venue)
+        : validationSchema.cast(),
+    [state.detailsPage, venueId]
+  );
+
+  const queryParams = useQuery();
+  const parentIdQuery = queryParams.get("parentId");
+
+  const dispatch = useDispatch();
+  const { sovereignVenueId, sovereignVenue } = useSovereignVenue({ venueId });
+
+  const {
+    watch,
+    formState,
+    register,
+    setValue,
+    control,
+    handleSubmit,
+    errors,
+    setError,
+  } = useForm<FormValues>({
+    mode: "onSubmit",
+    reValidateMode: "onChange",
+    validationSchema: validationSchema,
+    validationContext: {
+      template: state.templatePage?.template,
+      editing: !!venueId,
+    },
+    defaultValues: {
+      ...defaultValues,
+      logoImageUrl: defaultValues?.logoImageUrl ?? DEFAULT_VENUE_LOGO,
+      bannerImageUrl: defaultValues?.bannerImageUrl ?? DEFAULT_VENUE_BANNER,
+      parentId: parentIdQuery ?? defaultValues?.parentId ?? "",
+    },
+  });
+  const { user } = useUser();
+  const history = useHistory();
+  const { isSubmitting } = formState;
+
+  const [formError, setFormError] = useState(false);
+
+  //register the icon position data
+  useEffect(() => {
+    register("placement");
+  }, [register]);
+
+  const placementDivRef = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    const clientWidth = placementDivRef.current?.clientWidth ?? 0;
+    const clientHeight = placementDivRef.current?.clientHeight ?? 0;
+
+    placementDivRef.current?.scrollTo(
+      (state?.detailsPage?.venue.placement?.x ?? 0) - clientWidth / 2,
+      (state?.detailsPage?.venue.placement?.y ?? 0) - clientHeight / 2
+    );
+  }, [state]);
+
+  // @debt refactor this to split it into more manageable chunks, most likely with some things pulled into the api/* layer
+  // @debt refactor this to use useAsync or useAsyncFn as appropriate
+  const onSubmit = useCallback(
```

Comment:
[OOS] We're working to make the entire form obsolete with the Admin v3 because of the large volume of refactoring needed. This is just a stop gap for the Admin v1 not displaying feedback on error.

Reply:
[Note]
Lines in this file show up as added due to the rename of it.
Most of them have remained unchanged from their previous iteration.
Only minimal changes made in the actual code due to the display of errors.